polaris-react/src/components/IndexTable/IndexTable.stories.tsx

```diff
@@ -3929,3 +3933,1346 @@ export function WithSubHeaders() {
     </LegacyCard>
   );
 }
+
+export function WithNestedRows() {
```

Comment:
To add this example to the website, you'll need to add a file for it in `polaris.shopify.com/pages/examples` (duplicate and existing file, rename it, and replace the code with this story's code).

Reply:
Yes, you can run `yarn build && yarn turbo run dev --filter=polaris.shopify.com` and it will build the packages and site and then open localhost:3000 for you in a new tab/window when the build is successful.